src/wesnoth.cpp

```diff
@@ -941,7 +941,7 @@ static void restart_process(const std::vector<std::string>& commandline)
 #define error_exit(res) return res
 #endif
 
-#ifdef __APPLE__
+#ifdef __IPHONEOS__
```

Comment:
You could do something like what's done for the MSVC project, where the project file is in the repository but the dependencies are in a separate repository.

It's not exactly _important_ to have the project file committed though, so if you prefer it separate I guess I won't push it.